modules/custom/express_site_metrics/classes/express_site_metrics_token_handler.php

```diff
@@ -0,0 +1,513 @@
+<?php
+
+/**
+ * @file
+ * Code needed to work with Json Web Tokens.
+ */
+
+/**
+ * Class for working with Json Web Tokens.
+ */
```

Comment:
Ok. I added a comment in TokenHandler classes description.

You are correct, I think, about difficulty regarding updates and changes for security. I will look into a way to handle this. I am afraid that the solution we choose will include copy/pasting and editing code so that it will work in our D7/express codebase, which does not use PSR OOP namespaces, autoload ect that many Composer packages make use of.

Reply:
I 100% expect a copy/paste answer for getting code on the server, I am wondering if there is a way to leverage github automation for the alerting part when there is a security issue.